lucene/queries/src/java/org/apache/lucene/queries/intervals/IntervalIterator.java

```diff
@@ -82,6 +82,11 @@ public int width() {
   /**
    * Advance the iterator to the next interval
    *
+   * Should not be called after {@link DocIdSetIterator#NO_MORE_DOCS} is returned by other methods
+   * if that's the case in some existing code, please consider opening an issue
```

Comment:
This is a new sentence -- maybe add `.` at end of previous one and capitlize `If`?